### PR TITLE
Updated usercode API

### PR DESCRIPTION
**Pull request by imilicevic7**
Changes approved by Diaa.
+
Fixed annoying Z-Wave misspelling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -500,19 +500,20 @@ Detailed specification is avaliable on zwave-ad repo under docs folder.
...
+in   | cmd.usercode.set               | str_map    | 
+in   | cmd.usercode.get_slot          | str_map    | Requesting specific slot {"slot": "3"}
 out  | evt.usercode.access_report     | str_map    |
 
+
 #### Service props
 
 Name             | Value example              | Description
 -----------------|----------------------------|-------------
-`sup_usercodes`  | ["pin", "rfid"]            | List of supported user code types
```
Comment (ditek) [inline README.md:513]: I believe we should keep those. I checked zigbee-ad and it uses `code_type` (pin/rfid), `user_status` (enabled/disabled) and `user_type` (master/unrestricted). I think that's where those came from.

File: README.md [hunk 2]
```diff
@@ -500,19 +500,20 @@ Detailed specification is avaliable on zwave-ad repo under docs folder.
...
 Type | Interface                      | Value type | Description
 -----|--------------------------------|------------|------------------
-in   | cmd.usercode.clear             | str_map    |
-in   | cmd.usercode.clear_all         | null       |
-in   | cmd.usercode.get               | str_map    |
-in   | cmd.usercode.set               | str_map    |
+in   | cmd.usercode.clear             | str_map    | Delete specific slot
+in   | cmd.usercode.clear_all         | null       | Delete all users from all medium
+in   | cmd.usercode.get               | null       | Will return all slots for all medium
+in   | cmd.usercode.set               | str_map    |
```
Comment (ditek) [inline README.md:506]: Could you add what fields in the string map are expected?

File: README.md [hunk 3]
```diff
@@ -500,19 +500,20 @@ Detailed specification is avaliable on zwave-ad repo under docs folder.
 
 Type | Interface                      | Value type | Description
 -----|--------------------------------|------------|------------------
-in   | cmd.usercode.clear             | str_map    |
-in   | cmd.usercode.clear_all         | null       |
-in   | cmd.usercode.get               | str_map    |
-in   | cmd.usercode.set               | str_map    |
+in   | cmd.usercode.clear             | str_map    | Delete specific slot
+in   | cmd.usercode.clear_all         | null       | Delete all users from all medium
+in   | cmd.usercode.get               | null       | Will return all slots for all medium
```
Comment (ditek) [inline README.md:505]: Could you add how the get response looks like?

File: README.md [hunk 4]
```diff
@@ -500,19 +500,20 @@ Detailed specification is avaliable on zwave-ad repo under docs folder.
...
+in   | cmd.usercode.set               | str_map    | 
+in   | cmd.usercode.get_slot          | str_map    | Requesting specific slot {"slot": "3"}
 out  | evt.usercode.access_report     | str_map    |
```
Comment (ditek) [inline README.md:508]: It's not clear when this message is sent or what it contains

**Review by ditek — CHANGES_REQUESTED** [review]

**Comment by bjhandeland** [comment]
@imilicevic7 can you please update based on review?

**Comment by bjhandeland** [comment]
@imilicevic7 bump

**Comment by bjhandeland** [comment]
@eternalamenity Can you check if this is still valid? If no, close the PR.

**Comment by BudzikM** [comment]
Good afternoon. Is the owner of this PR still alive ?

**Comment by bjhandeland** [comment]
Unlikely. @BudzikM closing the PR.

**Pull request closed without merging**